packages/config/src/bin/serve-chrome.ts

```diff
@@ -0,0 +1,164 @@
+      return 'podman';
+    }
+    if (execSync('which docker').toString().trim().length > 0) {
+      return 'docker';
+    }
+  } catch (error) {
+    throw new Error('No container runtime found');
+  }
+
+  throw new Error('No container runtime found');
+}
+
+async function getLatestCommits(): Promise<string> {
+  const { data } = await axios.get(`https://api.github.com/repos/${REPO_OWNER}/${REPO_NAME}/commits`, {
+    headers: {
+      Accept: 'application/vnd.github+json',
+      'X-GitHub-Api-Version': '2022-11-28',
+    },
+    params: {
+      per_page: 1,
+    },
+  });
+  if (data.length === 0) {
+    throw new Error('No commits for chrome found!');
+  }
+
+  const { sha } = data[0];
+  return sha.substring(0, 7);
+}
+
+type Deployment = {
+  resourceTemplates: {
+    name: string;
+    targets: {
+      ref: string;
+      namespace: {
+        $ref: string;
+      };
+    }[];
+  }[];
+};
+
+async function getProdRelease() {
+  try {
+    if (!fs.existsSync(chromeDeploymentConfig.tarTarget)) {
+      fs.mkdirSync(chromeDeploymentConfig.tarTarget);
+    }
+    execSync(checkoutCommand, {
+      stdio: 'inherit',
+    });
+
+    const deployment = load(
+      fs.readFileSync(path.join(chromeDeploymentConfig.tarTarget, chromeDeploymentConfig.deployFile), {
+        encoding: 'utf-8',
+      })
+    ) as Deployment;
+    const chromeProd = deployment.resourceTemplates
+      .find((template) => {
+        return template.name === 'insights-chrome';
+      })
+      ?.targets.find((target) => {
+        return target.namespace.$ref.includes('prod-frontends');
+      });
+    if (!chromeProd) {
+      throw new Error('Unable to find chrome prod deployment configuration.');
+    }
+
+    return chromeProd.ref.substring(0, 7);
+  } catch (error) {
+    fecLogger(LogType.error, error);
+    fecLogger(LogType.warn, 'Unable to find chrome prod deployment! Falling back to latest image.');
+    return getLatestCommits();
+  }
+
+  // const { data: deploymentYaml } = await axios.get(deploymentSource);
+  // console.log(deploymentYaml);
+}
+
+function pullImage(tag: string) {
+  execSync(`${execBin} pull quay.io/cloudservices/insights-chrome-frontend:${tag}`, {
+    stdio: 'inherit',
+  });
+}
+
+function startServer(tag: string) {
+  try {
+    execSync(`${execBin} stop fec-chrome-local`, {
+      stdio: 'inherit',
+    });
+    execSync(`${execBin} rm fec-chrome-local`, {
+      stdio: 'inherit',
+    });
+  } catch (error) {
+    fecLogger(LogType.info, 'No existing chrome container found');
+  }
+  const child = spawn(
+    `${execBin} run -p ${EXPOSED_PORT}:${CONTAINER_PORT} --name fec-chrome-local quay.io/cloudservices/insights-chrome-frontend:${tag}`,
+    [],
+    {
+      stdio: 'ignore',
+      shell: true,
+    }
+  );
+  return child;
+}
+
+function copyIndex(path: string, isPreview = false) {
+  const copyCommand = `docker cp fec-chrome-local:/opt/app-root/src/build/${isPreview ? 'preview' : 'stable'}/index.html ${path}`;
```

Comment:
If both podman and docker are supported, should this maybe first check which runtime to use? (Not everyone might have installed the docker compatibility extension(

Reply:
Yup I'll fix that. Thanks